docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
```

Comment:
I think we should at least use `'3'` here to have all new features available once we need them.

I've also added a PR for the linked page.

Reply:
It doesn't change anything (newer docker-compose just interpretes it).
I just took it from here tbh:
https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/RenderingDocs/RenderWithDockerCompose.html#render-with-docker-compose

Reply:
I know, but why use TYPO3v8 when you can use TYPO3v9? ;-)

Also see https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument/pull/151

Reply:
ok